src/game_api/script/usertypes/level_lua.cpp

```diff
@@ -1749,8 +1749,8 @@ void register_usertypes(sol::state& lua)
     room_templates["waddler"] = {86};
     room_templates["ghistshop_backlayer"] = {87};
     room_templates["challange_entrance_backlayer"] = {90};
-    room_templates["blackmarket"] = {118};
-    room_templates["mothership_room"] = {125};
+    room_templates["blackmarket"] = {119};
+    room_templates["mothership_room"] = {126};
```

Comment:
It would be nice if those rooms also showed the correct name with the "Draw Gridlines" room names in overlunky, not necessary in this PR though